src/arkhamdb/DeckImporterMain.ttslua

```diff
@@ -224,6 +226,27 @@ function maybeAddInvestigatorCards(deck, slots)
   end
 end
 
+function maybeAddCustomizeUpgradeSheets(slots, configuration)
+  local allCardsBag = getObjectFromGUID(configuration.card_bag_guid)
+  for cardId, _ in pairs(slots) do
+    -- upgrade sheets for customizable cards
+    local upgradesheet = allCardsBag.call("getCardById", { id = cardId .. "-c" })
+    if upgradesheet ~= nil then
+      slots[cardId.."-c"] = 1
+    end
+  end
+end
+
+function maybeAddSummonedServitor(slots)
+  for cardId, cardCount in pairs(slots) do
+    -- spawn additional minicard for 'Summoned Servitor'
+    if cardId == "09080" then
+      slots["09080-m"] = cardCount
```

Comment:
I don't think we need the cardCount because you only ever need one mini-card per player since "Summoned Servitor" is unique (and wow this makes the changing of the upkeep function to flip your own servitor way easier 🤣)

Reply:
Huh.  I actually changed this to just be 1.  Must have missed it in an update.  And turns out all my doc changes got lost too.  Welll...  boo.